md_api/src/main/java/com/wipro/ats/bdre/GetLineageQueryByProcessId.java

```diff
@@ -44,9 +48,6 @@ public GetLineageQueryByProcessId() {
         acbFactory.autowireBean(this);
     }
 
-    private static final String[][] PARAMS_STRUCTURE = {
-            {"pid", "sub-process-id", " Process id whose lineage query to be extracted"},
-    };
 
     public List<LineageQuery> execute(String[] params) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Add the "@Override" annotation above this method signature [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3AS1161)